step-plans/step-plans-parser/src/main/java/step/repositories/parser/annotated/DefaultDescriptionStepParser.java

```diff
@@ -440,7 +440,7 @@ public static void echo(ParsingContext parsingContext, String expression) {
 		parsingContext.addArtefactToCurrentParent(result);
 	}
 	
-	@Step("Check (.*)$")
+	@Step(value="Check (Expression *=.*)$", priority=2)

```

Comment:
- This expression requires exactly one space between "Check" and "Expression" and might break some existing plans having more than one space.
- Keyvalue.g4 considers additional chars as space: [ \u00A0\t\r\n]. In order not to break existing expression I would also allow them as space

Would therefore recommend: Check[ \u00A0\t\r\n]+(Expression[ \u00A0\t\r\n]=.*)$

Reply:
Thanks, this indeed would have caused regressions, I updated as suggested.